src/20241108_AaveV3EthereumLido_AutomatedAGRSActivation/AutomatedAGRSActivation.md

```diff
@@ -0,0 +1,55 @@
+
+Regarding this pilot integration of automated AGRS with Chaos Labs new Edge infrastructure, it has the following rationale:
+
+- Whenever aligned with other strategic and technical aspects, Aave should benefit from infrastructure built by service providers, in this case, Chaos Labs.
+- This pilot will be very non-invasive and with only positive outcomes, as the AGRS layer will guarantee that only acceptable interest rate recommendations will reach the Aave protocol, while ideally having a way more dynamic rate.
+
+## Specification
+
+The automated AGRS will use another instance of AGRS (exactly the same codebase as the other model), but with the following constraints:
+
+- This instance will only have configurable rate-related parameters: base variable borrow rate, Slope 1, and Slope 2 and uOptimal (Kink).
+- Recommendations of these parameters will be submitted to [RiskOracle](https://github.com/ChaosLabsInc/risk-oracle/blob/main/src/RiskOracle.sol) smart contract, from the Edge off-chain infrastructure.
+- Between the risk oracle smart contract and the AGRS contract, there will be a very thin middleware [AaveStewardsInjector](https://etherscan.io/address/0x834a5aC6e9D05b92F599A031941262F761c34859), which will have the following logic:
+  - Will take recommendations from the Edge Risk Oracle side and propagate them to the AGRS contract.
+  - Enforce that only the WETH asset can be acted upon.
+  - Given the protections (percentage constraints and time delay) on the AGRS side and that it is an assumption that risk recommendation will be the time correct on the Edge Risk Oracle, the propagation will be permissionless.
+
+The [AaveStewardsInjector](https://etherscan.io/address/0x834a5aC6e9D05b92F599A031941262F761c34859) middle-ware contract will run on Chainlink Automation, and similar to other Aave Robots will be registered using the [AaveCLRobotOperator](https://etherscan.io/address/0x1cDF8879eC8bE012bA959EB515b11008E0cb6323) contract with 1500 LINK from the Ethereum Collector.
+
+The new instance of the [RiskSteward](https://etherscan.io/address/0x81aFd0F99c2Afa2f2DD7E387c2Ef9CD2a29b6E1A) will be given the RiskAdmin role with the following method: `ACL_MANAGER.addRiskAdmin()`
+
+As suggested by Chaos Labs, the new automated AGRS system will be configured with the following params:
+
+| Parameter                 | Percent change allowed | minimumDelay |
+| ------------------------- | ---------------------- | ------------ |
+| Base Variable Borrow Rate | 0.5% (absolute change) | 1 day        |
+| Slope 1                   | 0.5% (absolute change) | 1 day        |
+| Slope 2                   | 5% (absolute change)   | 1 day        |
+| Optimal Point (Kink)      | 3% (absolute change)   | 1 day        |
+
+## References
```

Comment:
Let's add more explicit links to:
- The injector
- The Edge oracle
- The EdgeRiskSteward